source/FreeRTOS_DNS.c

```diff
@@ -48,6 +48,9 @@
 #include "FreeRTOS_DHCP.h"
 #include "NetworkBufferManagement.h"
 #include "FreeRTOS_Routing.h"
+#if ( ipconfigUSE_IPv6 != 0 )
+    #include "FreeRTOS_IPv6_Sockets.h"
+#endif
```

Comment:
Do we really need the ipconfigUSE_IPv6 flag. This flag will be used for build seperation. I know historically this flag had varied interpretaions, but now this is only for V4/V6 build separation and unit-tests do not contribute to final release binary size and hence build separation does not apply to unit tests.

Reply:
Removed